resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/ResourceBuilder.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.resteasy.spi;
+
+import org.jboss.resteasy.spi.metadata.ResourceClass;
+import org.jboss.resteasy.spi.metadata.ResourceConstructor;
+
+import java.lang.reflect.Method;
+
+public interface ResourceBuilder
```

Comment:
I guess that these classes shouldn't be used by RESTEasy users. But this PR change original non-abstract ResourceBuilder class without non-static method to interface. But original ResourceBuilder class was public and had public static methods and inner static classes. @asoldano WDYT? Is it ok to change it?